src/screens/TokenDetail.js

```diff
@@ -0,0 +1,685 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import $ from 'jquery';
+import tokens from '../utils/tokens';
+import ModalConfirm from '../components/ModalConfirm';
+import ModalEditToken from '../components/ModalEditToken';
+import ModalPin from '../components/ModalPin';
+import QRCode from 'qrcode.react';
+import { CopyToClipboard } from 'react-copy-to-clipboard';
+import HathorAlert from '../components/HathorAlert';
+import { connect } from "react-redux";
+import ReactLoading from 'react-loading';
+import hathorLib from '@hathor/wallet-lib';
+
+const mapStateToProps = (state) => {
+  return {
+    historyTransactions: state.historyTransactions,
+  };
+};
+
+
+/**
+ * Screen to manage a token. Mint, melt, edit, unregister, configuration string.
+ *
+ * @memberof Screens
+ */
+class TokenDetail extends React.Component {
```

Comment:
It seems this class is doing too much. Maybe we should refactor and move the operations to specific components. What do you think?

Reply:
@msbrogli I've made a big refactor that creates a component for each of the actions. Can you see if it's ok now?

Reply:
Great. Much easier to review and to understand. I think you should move each of them to a different file. You can put them in `components/tokens/FILE.js`, e.g. `mint.js`, `melt.js`, `destroy.js`, `delegate.js`, etc.

Reply:
Done